sys/net/gnrc/link_layer/lorawan/gnrc_lorawan_region.c

```diff
@@ -14,10 +14,13 @@
  */
 #include "net/gnrc/lorawan/region.h"
 
-#define GNRC_LORAWAN_DATARATES_NUMOF 6
+#define GNRC_LORAWAN_DATARATES_NUMOF (6U)
 
-static uint8_t dr_sf[GNRC_LORAWAN_DATARATES_NUMOF] = { LORA_SF12, LORA_SF11, LORA_SF10, LORA_SF9, LORA_SF8, LORA_SF7 };
-static uint8_t dr_bw[GNRC_LORAWAN_DATARATES_NUMOF] = { LORA_BW_125_KHZ, LORA_BW_125_KHZ, LORA_BW_125_KHZ, LORA_BW_125_KHZ, LORA_BW_125_KHZ, LORA_BW_125_KHZ };
+static uint8_t dr_sf[GNRC_LORAWAN_DATARATES_NUMOF] =
+{ LORA_SF12, LORA_SF11, LORA_SF10, LORA_SF9, LORA_SF8, LORA_SF7 };
+static uint8_t dr_bw[GNRC_LORAWAN_DATARATES_NUMOF] =
+{ LORA_BW_125_KHZ, LORA_BW_125_KHZ, LORA_BW_125_KHZ, LORA_BW_125_KHZ,
```

Comment:
same here, put 3 values per line

```
static uint8_t dr_bw[GNRC_LORAWAN_DATARATES_NUMOF] = {
    LORA_BW_125_KHZ, LORA_BW_125_KHZ, LORA_BW_125_KHZ,
    LORA_BW_125_KHZ, LORA_BW_125_KHZ, LORA_BW_125_KHZ
};
```

Reply:
looks IMHO nicer and more readable

Reply:
ah damn, but that came from the uncrustify, right?!

mhm, still mine looks better 😬